### DISCUSSION TITLE
Errors when trying to install udm-boot on UDM Pro

### DISCUSSION THREAD

**Discussion by finnhagard** [unifi-utilities/unifi-common#636] [General]
I'm trying to install udm-boot on my UDM pro but it's just erroring everytime. 

root@DreamMachinePro:~# dpkg -i udm-boot_1.0.6_all.deb
Selecting previously unselected package udm-boot.
(Reading database ... 53669 files and directories currently installed.)
Preparing to unpack udm-boot_1.0.6_all.deb ...
/var/lib/dpkg/tmp.ci/preinst: 19: /sbin/ssh-proxy: not found
dpkg: error processing archive udm-boot_1.0.6_all.deb (--install):
 new udm-boot package pre-installation script subprocess returned error exit status 127
/var/lib/dpkg/tmp.ci/postrm: 24: /sbin/ssh-proxy: not found
dpkg: error while cleaning up:
 new udm-boot package post-removal script subprocess returned error exit status 127
Errors were encountered while processing:
 udm-boot_1.0.6_all.deb

**Comment by m4n63** [comment]
There is a newer .deb available, and you can also try this that I have submitted as a PR.
https://github.com/m4n63/unifios-utilities/raw/refs/heads/main/on-boot-script-2.x/udm-boot-2x_1.0.1+nmu1_all.deb
